chart/templates/98-nginx-proxy.yml

```diff
@@ -81,10 +81,19 @@ spec:
               name: process-env
           - configMapRef:
               name: ssl-env
+          - configMapRef:
+              name: opensearch-env
+          # - configMapRef:
```

Comment:
As I said above, the `NGINX_DASHBOARDS_PREFIX` and `NGINX_DASHBOARDS_PROXY_PASS` are actually used by this container.

Reply:
Moved them to nginx-env configmap